airbyte-integrations/connectors/source-stripe/source_stripe/source.py

```diff
@@ -105,6 +160,7 @@ def streams(self, config: Mapping[str, Any]) -> List[Stream]:
             "account_id": config["account_id"],
             "start_date": config["start_date"],
             "slice_range": config["slice_range"],
+            "call_budget": self.get_api_call_budget(config),
```

Comment:
Should this be `api_budget` instead of `call_budget`? I fear this code would lead to `unexpected keyword argument` See: https://github.com/airbytehq/airbyte/blob/7d5026405a2b6ecfd8f0fa99166f91c8fa93db82/airbyte-cdk/python/airbyte_cdk/sources/streams/http/http.py#L42